packages/react-composites/src/composites/common/AvatarPersona.tsx

```diff
@@ -87,3 +82,58 @@ export const AvatarPersona = (props: AvatarPersonaProps): JSX.Element => {
     />
   );
 };
+
+/**
+ * Hook to override the avatar persona data of the users
+ * @private
+ */
+export const useDataProvider = (
```

Comment:
Let's rename this to better reflect what it does: `useCustomAvatarPersonaData` or some such.

Also, should this move to a utils module. We'd want to use it each time we need custom avatar data anywhere.